src/Tizen.Log/Tizen/Log.cs

```diff
@@ -15,11 +15,130 @@
  */
 
 using System;
+using System.Text;
 using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
 using System.ComponentModel;
 
 namespace Tizen
 {
+    internal static class LogPrinter
+    {
+        private static char[] sep = { '\\', '/' };
+        private static byte[] fmt1 = Encoding.UTF8.GetBytes("%s");
+        private static byte[] fmt2 = Encoding.UTF8.GetBytes("%s: %s(%d) > %s");
+
+        private static unsafe void _Print(Interop.Dlog.LogID log_id, Interop.Dlog.LogPriority priority, string tag, string message, string file, string func, int line)
+        {
+            int tagByteLength = Encoding.UTF8.GetMaxByteCount(tag.Length);
+            Span<byte> tagByte = tagByteLength < 1024 ? stackalloc byte[tagByteLength + 1] : new byte[tagByteLength + 1];
+
+            int messageByteLength = Encoding.UTF8.GetMaxByteCount(message.Length);
+            Span<byte> messageByte = messageByteLength < 1024 ? stackalloc byte[messageByteLength + 1] : new byte[messageByteLength + 1];
+
+            fixed (char* pTagChar = &MemoryMarshal.GetReference(tag.AsSpan()))
```

Comment:
```suggestion
            fixed (char* pTagChar = tag)
```
이렇게는 안되나요?
https://github.sec.samsung.net/dotnet/corefx/blob/73f892e50b58688da3ba79099dd362ec945e57f0/src/System.Console/src/System/ConsolePal.Unix.cs#L1375

여기 래퍼런스 할수 있는 코드를 string을 바로 char*로 대입하는것 같습니다.

Reply:
수정하였습니다. 감사합니다.